cmd/system-probe/modules/network_tracer.go

```diff
@@ -149,24 +149,22 @@ func writeConnectionsTogRPC(marshaler encoding.Marshaler, cs *network.Connection
 		return nil, err
 	}
 
-	log.Tracef("/connections: %d connections, %d bytes", len(cs.Conns), len(buf))
+	log.Tracef("/GetConnections: %d connections, %d bytes", len(cs.Conns), len(buf))
```

Comment:
```suggestion
	log.Tracef("GetConnections: %d connections, %d bytes", len(cs.Conns), len(buf))
```

Reply:
it makes sense to have `/connection` as a prefix the a trace log in the handler of the `/connections` endpoint in the http server, in grpc server is makes less sense